test/functional/terminal/scrollback_spec.lua

```diff
@@ -416,7 +440,12 @@ describe("'scrollback' option", function()
     -- Terminal job data is received asynchronously, may happen before the
     -- 'scrollback' option is synchronized with the internal sb_buffer.
     command('sleep 100m')
-    feed_data('for i in $(seq 1 40); do echo "line$i"; done\n')
+    if iswin() then
+      nvim('command',
```

Comment:
Can't feed_data be used ? It does jobsend on the child instance in the terminal.